src/application/globals/writeCollectionPreRequestScripts.ts

```diff
@@ -19,11 +20,18 @@ export const writeCollectionPreRequestScripts = (
     } as EventDefinition
   }
 
+  const scriptContents = scripts.map(src => {
+    if (src.startsWith('file:')) {
+      return fs.readFileSync(src.replace('file:', ''), { encoding:'utf8', flag:'r' })
```

Comment:
Potentially, it would be safer to wrap this in a try {} catch, and throw an error if the file does not exist.

```js
try {
      // your scriptContents logic
    } catch (err) {
      console.error('\x1b[31m', 'Config pre-request script file error - no such file or directory "' + src + '"')
      if (options.verbose >= 1) {
        console.error(err)
      }
      process.exit(1)
    }
```